### ISSUE TITLE
An Internal Error Has Occurred - Importing from Path Shortcut

### ISSUE THREAD

**Issue by finitenamespace** [burtonator/polar-bookshelf#560]
I'm receiving "An Internal Error Has Occurred" when importing from a file location using a shortcut.

File > Import > Navigate to the location using the shortcut on the left-hand pane = Error.
If I do the same, but navigate to the full path by starting in C:\ and working my way to the NextCloud folder, it works.

Running:
Windows 10
Polar 1.8.0

Log output:
2019-01-16T06:32:08.435Z: [error] [1.8.0] [FileImportController.js] Failed to import file: \\hostname\c$\Users\user\Nextcloud\No Work Sync\Books\SICP - Structure & Interpretation of Computer Programs.pdf: [{"name":"MissingPDFException","message":"Missing PDF \"/c$/Users/user/Nextcloud/No Work Sync/Books/SICP - Structure & Interpretation of Computer Programs.pdf\"."}]

Unfortunately it seems like Polar only logs errors, not successes, so I can't show you the difference between how Polar is interpreting the two paths to the same location.

**Comment by burtonator** [comment]
Yes.. I might have to implement some changes so that the paths are always logged.

**Comment by finitenamespace** [comment]
Might help narrow down the problem 👍
